test/jdk/java/nio/charset/Charset/Default.java

```diff
@@ -1,36 +0,0 @@
-/*
```

Comment:
This class is used in `DefaultCharsetTest` so don't remove it.

Reply:
Fixed, thank you